docs/src/user/user-intro.adoc

```diff
@@ -103,41 +103,41 @@ AXIS:: <<cha:axis-gui,AXIS>>, the standard keyboard GUI interface. This is also
   Configuration Wizard is used to create a desktop icon launcher:
 
 [[fig:axis-graphical-interface]]
-.AXIS graphical interface
+//.AXIS graphical interface
 image::../gui/images/axis.png["Axis, the standard keyboard GUI interface",align="center"]
 
 Touchy:: <<cha:touchy-gui,Touchy>>, a touch screens GUI:
 
 [[fig:touchy-graphical-interface]]
-.Touchy graphical interface
+//.Touchy graphical interface
 image::../gui/images/touchy.png["Touchy, a touch screen GUI",align="center"]
 
 Gscreen:: <<cha:gscreen,Gscreen>>, a user-configurable touch screen GUI:
 
 [[fig:gscreen-graphical-interface]]
-.Gscreen graphical interface
+//.Gscreen graphical interface
 image::../gui/images/gscreen-mill.png["Gscreen, a configurable base touch screen GUI",align="center"]
 
 GMOCCAPY:: <<cha:gmoccapy,GMOCCAPY>>, a touch screen GUI based on Gscreen. GMOCCAPY is also designed to work equally
   well in applications where a keyboard and mouse are the preferred methods of controlling the GUI:
 
 [[fig:gmoccapy-graphical-interface]]
-.GMOCCAPY graphical interface
+//.GMOCCAPY graphical interface
 image::../gui/images/gmoccapy_3_axis.png["gmoccapy, a touch screen GUI based on Gscreen",align="center"]
 
 NGCGUI:: <<cha:ngcgui,NGCGUI>>, a subroutine GUI that provides wizard-style programming of G code. NGCGUI may be
   run as a standalone program or embedded into another GUI as a series of tabs. The following screen shot
   shows NGCGUI embedded into Axis:
 
 [[fig:ngcgui-graphical-interface-into-axis]]
-.NGCGUI graphical interface integrated into Axis
+//.NGCGUI graphical interface integrated into Axis
 image::../gui/images/ngcgui.png["NGCGUI graphical interface integrated into Axis",align="center"]
```

Comment:
And for those I would suggest

```suggestion
.NGCGUI graphical interface integrated into Axis
image::../gui/images/ngcgui.png[align="center"]
```
or
```suggestion
.NGCGUI graphical interface integrated into Axis
//image::../gui/images/ngcgui.png["NGCGUI graphical interface integrated into Axis",align="center"]
image::../gui/images/ngcgui.png[,align="center"]
```
or
```suggestion
.NGCGUI graphical interface integrated into Axis
image::../gui/images/ngcgui.png["The NGCGUI graphical interface integrated into Axis",align="center"]
```
(slighly modifiyng the alt text should fix the po4a problem, right?)

Reply:
> I recall to have shifted about ~20 or so captions to alt-texts. In part this could be motivated by the translation having it as an alt. Lacking the style guide, I considered it reasonable to have everything equal for all images. And frankly, since we need alt texts anyway, I am happy with how I now did it and suggest we come up with decent captions for all images as a To-Do item. I prefer a clear-cut out-commenting over a partial rewrite when what is truly missing is a real caption.

I had a closer look and these captions are introduced by you or Jérémie during this process. So in this case it is fine for me.
But why don't keep them in cases where they don't make problems (caption != alt) ?

Reply:
caption != alt != header - please also check that.
And, the translation needs the same caption or alt or header.
There may be cases in which I have anticipated a problem and already commented something out prior to the error occurring. But otherwise, if there was no problem, then I would not have touched the file in the first place.

Reply:
ah ok a header with the same name some lines before...

Reply:
Lucky me :)